util/README.md

```diff
@@ -1,38 +1,35 @@
-# Convert Excel sheet to JSON
+## Convert Excel Sheet to JSON
 
-## Usage
+### Overview
 
-`convert_xlsx_to_json.js` is used to convert excel to JSON.
-`convert_json_to_data.js` has the transformation code to adhere to the required JSON format.
+The `convert_xlsx_to_json.js` script converts an Excel file to JSON format. The `convert_json_to_data.js` script transforms the JSON data to adhere to the required format.
 
-_`data/all_data.json` and `data/all_data.json` will be modified as per the new excel_
+### Usage
 
-### Note
-
--   Make sure excel file name is `report_<campus>.xlsx`
-
--   The sheet name in the file MUST be `Sheet 1`
+-   The scripts modify `data/all_data.json` as per the new Excel data.
+-   Ensure the Excel file is named in the format `report_<campus>.xlsx`.
+-   The sheet name must be `Sheet 1`.
 
--   Excel sheet should ONLY have the following columns headers (case sensitive):
+### Requirements
 
-    **CODE**
+The Excel sheet should contain the following column headers (case sensitive):
 
-    **TITLE**
+| CODE | TITLE | TYPE | CREDITS | SLOT | FACULTY | VENUE |
+| ---- | ----- | ---- | ------- | ---- | ------- | ----- |
 
-    **TYPE**
+### Instructions
 
-    **VENUE**
+1. Use the `convert_xlsx_to_json.js` script to convert the Excel file to JSON.
+2. Use the `convert_json_to_data.js` script to transform the JSON data into the required format.
 
-    **FACULTY**
-
-    **CREDITS**
+### Note
 
-    **SLOT**
+Both of these scripts can be executed using the command `yarn run convert`.
```

Comment:
Can you put this in a blockquote as shown in the code [in my earlier message](https://github.com/vatz88/FFCSonTheGo/pull/251/files#r1663577111), this doesn't have to be a separate heading.